libs/datamodel/connectors/mongodb-datamodel-connector/src/lib.rs

```diff
@@ -120,65 +54,17 @@ impl Connector for MongoDbDatamodelConnector {
     }
 
     fn validate_model(&self, model: ModelWalker<'_>, errors: &mut Diagnostics) {
-        for field in model.scalar_fields() {
-            Self::validate_auto(field, errors);
-            Self::validate_dbgenerated(field, errors);
-            Self::validate_array_native_type(field, errors);
-        }
+        validations::id_must_be_defined(model, errors);
 
         if let Some(pk) = model.primary_key() {
```

Comment:
could the `id_must_be_defined()` be in the `else` of this `if let`? It may make things simpler.

Reply:
It was, but I'd like validations to stay out of this file...